recipe/meta.yaml

```diff
@@ -10,13 +10,12 @@ source:
   sha256: 1921f26d2e1daf9132da3c432e2fd02093ecaedf846e65d7679ddf868c7289c4
 
 build:
-  noarch: generic
```

Comment:
Out of curiosity, why can we not be noarch anymore?

Reply:
I assumed that the contents of the cmake configuration would be platform dependent. I will check that.

Reply:
The cmake configuration files are identical, but the cxx compiler requirement that cmake imposes causes `conda-smithy` to generate separate Linux, Mac, and Windows builds. I reinstated `noarch: generic` by patching `CMakeLists.txt` so that the CXX compiler is no longer required to install cereal.